src/sqlfluff/core/rules/base.py

```diff
@@ -99,6 +99,14 @@ def __init__(
         # store a description_override for later
         self.description = description
 
+    def __repr__(self):
+        if not self.anchor:
+            return "LintResult(<empty)"
+        fix_coda = f"+{len(self.fixes)}F" if self.fixes else ""
```

Comment:
Is the capital `F` at the end of the string intentional?

Reply:
Yeah it's supposed to be short for "fixes". I'll add a note.